bin/condense_sdrf.pl

```diff
@@ -310,7 +310,7 @@ sub copy_idf_from_ae {
     my ( $args, $idfFile ) = @_;
     $logger->info( "Copying IDF from $idfFile ..." );
     my $outputDir = $args->{ "output_directory" };
-    `cp $idfFile $outputDir`;
+    `cp -f $idfFile $outputDir`;
```

Comment:
What is the point of this?

Reply:
This is to mitigate the following error in rule `get_magetab_for_experiment`
```
cp: cannot create regular file '[ACC]/[ACC].idf.txt': Permission denied
```
for some diff proteomics accessions.